test/actions.test.ts

```diff
@@ -185,9 +185,12 @@ describe("Test Custom SDK Logic", () => {
   describe("(get|set)buyNowPrice", () => {
     it("runs as expected", async () => {
```

Comment:
nit: instead of having one vague "runs as expected" test, it might be better to have a separate test for this scenario.
E.g. "returns undefined when the owner has changed"